molgenis-data-import/src/main/java/org/molgenis/data/importer/emx/IntermediateParseResults.java

```diff
@@ -95,14 +96,22 @@ public void addAttributes(String entityTypeId, List<EmxAttribute> emxAttrs) {
     }
 
     if (!entityType.getLookupAttributes().iterator().hasNext()) {
-      Attribute ownIdAttr = entityType.getOwnIdAttribute();
-      if (ownIdAttr != null && ownIdAttr.isVisible()) {
-        ownIdAttr.setLookupAttributeIndex(lookupAttributeIndex++);
-      }
-      Attribute ownLabelAttr = entityType.getOwnLabelAttribute();
-      if (ownLabelAttr != null) {
-        ownLabelAttr.setLookupAttributeIndex(lookupAttributeIndex);
-      }
+      setDefaultLookupAttributes(entityType, lookupAttributeIndex);
+    }
+  }
+
+  public static void setDefaultLookupAttributes(EntityType entityType, int lookupAttributeIndex) {
+    Attribute ownIdAttr = entityType.getOwnIdAttribute();
+    if (ownIdAttr != null && ownIdAttr.isVisible()) {
+      ownIdAttr.setLookupAttributeIndex(lookupAttributeIndex);
+      lookupAttributeIndex++;
+    }
+    Attribute ownLabelAttr = entityType.getOwnLabelAttribute();
+    if (ownLabelAttr != null
+        && ownIdAttr != null
+        && !ownIdAttr.getName().equals(ownLabelAttr.getName())) {
+      ownLabelAttr.setLookupAttributeIndex(lookupAttributeIndex);
+      lookupAttributeIndex++;
```

Comment:
Sonar reports an error on this line: 
- Remove this useless assignment to local variable "lookupAttributeIndex"